plugins/main/bridge/bridge.go

```diff
@@ -101,6 +110,11 @@ func loadNetConf(bytes []byte, envArgs string) (*NetConf, string, error) {
 	if n.Vlan < 0 || n.Vlan > 4094 {
 		return nil, "", fmt.Errorf("invalid VLAN ID %d (must be between 0 and 4094)", n.Vlan)
 	}
+	var err error
+	if n.vlans, err = collectVlanTrunk(n.VlanTrunk); err != nil {
```

Comment:
Would it be more consistent with the rest of the code to use the scoped variable and not define err?

Reply:
We will still need to copy the value back to `n.vlans` later.
So I will consider to keep current code